### PR TITLE
privacy added

### PR DESCRIPTION
**Pull request by alkock**
Link zum index fehlt noch, müsste im Footer noch eingepflegt werden.

### CODE REVIEW

**Comment by Giom-fm** [comment]
> Link zum index fehlt noch, müsste im Footer noch eingepflegt werden.

Could you add it? Have a look at [menus.de.toml](https://github.com/Giom-fm/ff-eppendorf-website/blob/a5e0dea44ae52ef83851d59ada905d81e1afd008/config/_default/menus/menus.de.toml#L72)

I would say in this order: Impressum, Datenschutz, Kontakt.

**Comment by alkock** [comment]
I added the file to the config for the index. In the order above :)

**Review by Giom-fm — CHANGES_REQUESTED** [review]

File: content/de/datenschutz.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Datenschutzerklärung
+## Einleitung
+Mit der folgenden Datenschutzerklärung möchten wir Sie darüber aufklären, welche Arten Ihrer personenbezogenen Daten (nachfolgend auch kurz als "Daten“ bezeichnet) wir zu welchen Zwecken und in welchem Umfang verarbeiten. Die Datenschutzerklärung gilt für alle von uns durchgeführten Verarbeitungen personenbezogener Daten, sowohl im Rahmen der Erbringung unserer Leistungen als auch insbesondere auf unseren Webseiten, in mobilen Applikationen sowie innerhalb externer Onlinepräsenzen, wie z.B. unserer Social-Media-Profile (nachfolgend zusammenfassend bezeichnet als "Onlineangebot“).
+
+Die verwendeten Begriffe sind nicht geschlechtsspezifisch.
+
+Stand: 23. Januar 2023
+
+## Inhaltsübersicht
```
[the remark this answers is not in the document]
Comment (alkock) [reply content/de/datenschutz.md]: Removed the section entirely.

File: content/de/datenschutz.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+<li>Übersicht der Verarbeitungen</li>
+<li>Maßgebliche Rechtsgrundlagen</li>   
+<li>Sicherheitsmaßnahmen</li>
+<li>Löschung von Daten</li>
+<li>Bereitstellung des Onlineangebotes und Webhosting</li>
+<li>Änderung und Aktualisierung der Datenschutzerklärung</li>
+<li>Rechte der betroffenen Personen</li>
+<li>Begriffsdefinitionen</li>
+
+### Verantwortlicher
```
[the remark this answers is not in the document]
Comment (alkock) [reply content/de/datenschutz.md]: Used the same format as in impressum.

File: content/de/datenschutz.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+<li>Verantwortlicher: Als "Verantwortlicher“ wird die natürliche oder juristische Person, Behörde, Einrichtung oder andere Stelle, die allein oder gemeinsam mit anderen über die Zwecke und Mittel der Verarbeitung von personenbezogenen Daten entscheidet, bezeichnet.</li>
+<li>Verarbeitung: "Verarbeitung" ist jeder mit oder ohne Hilfe automatisierter Verfahren ausgeführte Vorgang oder jede solche Vorgangsreihe im Zusammenhang mit personenbezogenen Daten. Der Begriff reicht weit und umfasst praktisch jeden Umgang mit Daten, sei es das Erheben, das Auswerten, das Speichern, das Übermitteln oder das Löschen.</li>
+<br>
+<a href="https://datenschutz-generator.de/">Erstellt mit kostenlosem Datenschutz-Generator.de von Dr. Thomas Schwenke</a>
```
[the remark this answers is not in the document]
Comment (alkock) [reply content/de/datenschutz.md]: Replaced with markdown-link

**Comment by Giom-fm** [comment]
It seams that you deleted a lot of files. Did you rebased already?

File: content/de/datenschutz.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+2. Verantwortlicher:    
+ Als "Verantwortlicher“ wird die natürliche oder juristische Person, Behörde, Einrichtung oder andere Stelle, die allein oder gemeinsam mit anderen über die Zwecke und Mittel der Verarbeitung von personenbezogenen Daten entscheidet, bezeichnet.
+3. Verarbeitung:   
+ "Verarbeitung" ist jeder mit oder ohne Hilfe automatisierter Verfahren ausgeführte Vorgang oder jede solche Vorgangsreihe im Zusammenhang mit personenbezogenen Daten. Der Begriff reicht weit und umfasst praktisch jeden Umgang mit Daten, sei es das Erheben, das Auswerten, das Speichern, das Übermitteln oder das Löschen.
+<br>
```
Comment (Giom-fm) [inline content/de/datenschutz.md:114]: There is one left ;)

Comment (alkock) [reply content/de/datenschutz.md]: oops did a br fix :)

**Comment by alkock** [comment]
> It seams that you deleted a lot of files. Did you rebased already?

Should be resolved, I see just two files [here](https://github.com/Giom-fm/ff-eppendorf-website/pull/3/files)

**Review by Giom-fm — APPROVED** [review]

**Pull request merged**